ddtrace/contrib/flask_cache/tracers.py

```diff
@@ -54,9 +68,10 @@ def __trace(self, cmd):
             # set analytics sample rate
             s.set_tag(ANALYTICS_SAMPLE_RATE_KEY, config.flask_cache.get_analytics_sample_rate())
             # add connection meta if there is one
-            if getattr(self.cache, "_client", None):
+            client = _extract_client(self.cache)
```

Comment:
```suggestion
            client = _extract_client(self.cache, write)
```
?

Reply:
Good catch! The argument is used only for the Redis Sentinel backend which is unfortunately untested by our test suite and so it didn't catch the issue. I tried to add a Redis Sentinel test but it's not trivial and requires setting up multiple instances of Redis.